README.md

```diff
@@ -36,6 +36,8 @@ Browse and query your MongoDB databases both locally and in the cloud using [_sc
 - Enter your commands and/or comments, eg: `db.<collectionName>.find()`
 - IntelliSense (auto-completions) will be provided
 - Place the cursor inside the command to execute and press `CMD+"` (Mac) or `CTRL+"` (Windows and Linux) to see the results
+- To run all commands, press `CMD+:` or `Ctrl+:`
+- With codelens, you can perform the above actions without having to leave the editor pane

```

Comment:
I'm not sure this line adds anything meaningful.  Either way you don't have to leave the editor pane.